File: networking/src/main/java/ro/mpp2024/dto/EmployeeDTO.java
package ro.mpp2024.dto;

public class EmployeeDTO {
    private Integer id;
    private final String username;
    private final String password;
    private final Integer officeId;

    public EmployeeDTO(Integer id, String username, String password, Integer officeId) {
        this.id = id;
        this.username = username;
        this.password = password;
        this.officeId = officeId;
    }

    public Integer getId() {
        return id;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }

    public Integer getOfficeId() {
        return officeId;
    }

    @Override
    public String toString() {
        return "EmployeeDTO{" +
                "id=" + id +
                ", username='" + username + '\'' +
                ", password='" + password + '\'' +
                ", officeId=" + officeId +
                '}';
    }
}

File: csharpServer/networking/dto/EmployeeDTO.cs
namespace networking.dto;

public class EmployeeDTO
{
    public int? Id { get; set; }
    public string Username { get; }
    public string Password { get; }
    public int? OfficeId { get; }

    public EmployeeDTO(int? id, string username, string password, int? officeId)
    {
        Id = id;
        Username = username;
        Password = password;
        OfficeId = officeId;
    }

    public override string ToString()
    {
        return $"EmployeeDTO{{id={Id}, username='{Username}', password='{Password}', officeId={OfficeId}}}";
    }
}